### PR TITLE
Redundancy: Changed 'deco' to 'docs'

### PR DESCRIPTION
**Pull request by SACHIN-13**
in line-98: coalib/bearlib/aspects/Redundancy.py
Closes https://github.com/coala/coala/issues/3684

### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

**After you submit your pull request, DO NOT click the 'Update Branch' button**

### CODE REVIEW

**Comment by SACHIN-13** [comment]
Okay!

**Comment by RohanVB** [comment]
@SACHIN-13 adding to what the user above said, you'll have to put the entire link in place of the `3684` for the reference. Check [this](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) out.

**Comment by SACHIN-13** [comment]
i specified the link only!

**Comment by RohanVB** [comment]
My bad. I must've checked the PR body instead of the commit log.

**Comment by SACHIN-13** [comment]
the log shows CSSLint bear error. csslint not installed! couldnt find a way to do it!

File: coalib/bearlib/aspects/Redundancy.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ class UnreachableCode:
     Unreachable code, sometimes called dead code, is source code that
     can never be executed during the program execution.
     """
```
Comment (Adrianzatreanu) [inline coalib/bearlib/aspects/Redundancy.py]: commit shortlog: Redundency -> Redundancy

Comment (SACHIN-13) [inline coalib/bearlib/aspects/Redundancy.py]: The check still fails :(

**Comment by SACHIN-13** [comment]
I made a change from 'Fixes' -> 'Closes' as suggested earlier since it was not a bug that I fixed!

**Comment by jayvdb** [comment]
ack a6cddd9

**Comment by jayvdb** [comment]
@rultor merge

**Comment by SACHIN-13** [comment]
already did that! and now it is up-to-date.

File: coalib/bearlib/aspects/Redundancy.py [hunk 2]
```diff
@@ -95,7 +95,7 @@ class UnreachableCode:
...
-    class deco:
+    class docs:
```
Comment (Mixih) [inline coalib/bearlib/aspects/Redundancy.py]: isn't this going to break bears?

**Comment by SACHIN-13** [comment]
The build fails again and again. Cannot figure out why!

**Comment by sils** [comment]
please do a rebase and check if the tests pass then! Unapproving in the meantime due to testfail.

**Comment by SACHIN-13** [comment]
the log shows this messgae `ERROR: You must allocate 2 containers for the tests to run properly!`

**Comment by sils** [comment]
@SACHIN-13 you're running the build on a fork right? Configue circle to use two containers and make a new build please.

**Comment by sils** [comment]
hey, please use `cobot mark pending <pr URL>` if you restarted the build with two containers and it passes!

**Comment by Udayan12167** [comment]
ack https://github.com/coala/coala/pull/3685/commits/a31eca14e2af951d442e447f45e259f2c205eba7

**Comment by Udayan12167** [comment]
@rultor merge

**Pull request merged**